exaudfclient/base/exaudflib/test/script_data_transfer_objects_test.cpp

```diff
@@ -258,9 +258,3 @@ TEST_F(ConnectionInformationTest, copy)
     EXPECT_TRUE(ca.getPassword() == cb.getPassword());
     EXPECT_TRUE(ca.hasData() == cb.hasData());
```

Comment:
I know this did not change, but this looks really wrong to me! `ASSERT_TRUE(true)`

Likely something along the lines like this:
```cpp
    EXPECT_TRUE(ca.hasData());
    EXPECT_STREQ(ca.getPassword(), "password" );
```

would make more sense.

Reply:
Done